getting_started.ipynb

```diff
@@ -5,15 +5,17 @@
    "id": "bd00bd98-9b7e-4437-8518-90d25e69a00e",
```

Comment:
<p>change <code>variable for  You</code> to <code>variable. You</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jbusecke/tutorial_osm22_cmip6/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>